### ISSUE TITLE
Collisions are not working properly

### ISSUE THREAD

**Issue by mcrimson974** [shteeve3d/blender-wiggle-2#75]
Is anyone having Issues with Collisions?
On my end, when I enable collisions, it just SNAPS to the collider like a magnet. Tried on 4.1 and 4.2

**Comment by KaijuKoder2** [comment]
Yes, @mcrimson974, look at the other issues. Many discuss collisions problems. 

Note to @shteeve3d on his return: If you comment the script especially around the collisions I would be happy to pick that up. I was thinking of code that detects when a point is inside the mesh, then pulls it to the surface, and maintains length to the parent bone in the chain which should stop the bone stretching problem described in other issues.
